src/main/java/com/jcabi/http/mock/MkGrizzlyAdapter.java

```diff
@@ -118,8 +112,7 @@ public void service(final GrizzlyRequest request,
                         )
                     );
                     response.setStatus(answer.status());
-                    final byte[] body =
-                        answer.body().getBytes(MkGrizzlyAdapter.CHARSET);
+                    final byte[] body = answer.bodyBytes();
```

Comment:
@mkordas added a Test for the new functionality: see `MkContainerTest.answersBinary()`